src/careamics/dataset/dataset_utils/__init__.py

```diff
@@ -13,7 +15,12 @@
 ]
 
 
-from .dataset_utils import compute_normalization_stats, reshape_array
+from .dataset_utils import (
+    compute_normalization_stats,
```

Comment:
I think I'd prefer `from .data_utils.welford import ...`

For methods that are clearly part of a particular algorithm, I would like this to be clearly identified, even in the import!